.eslintrc.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
     "parser": "babel-eslint",
     "env": {
         "browser": true,
-        "es6": true
+        "es6": true,
+        jest: true
```

Comment:
Do we need any special plugins for babel or eslint to support this? Curious what this actually does.

Reply:
You shouldn't as far I am aware, this resolved a problem in my IDE (VsCode with the Jest and EsLin extensions) such that ESlint would stop syntax highlighting describe statements since it wasn't recognizing Jest. It should just be telling ESLint that it should be aware of the use of Jest in our Javascript.